prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,35 @@
+const number1 = 6,
+  number2 = 4,
+  product = 4 * 6;
+console.log(`value of product:`, product);
+console.log(`type of product:`, typeof product);
+const charge = 10,
+  payment = 3;
+let amountRemaining = charge - payment;
+console.log(`value of amountRemaining:`, amountRemaining);
+console.log(`type of amountRemaining:`, typeof amountRemaining);
+const tests = 80,
+  assignments = 95,
+  final = 83,
+  grade = (tests + assignments + final) / 3;
+console.log(`value of grade:`, grade);
+console.log(`type of grade:`, typeof grade);
+const firstName = `Daniel`,
+  lastName = `Yount`,
+  fullName = `${firstName} ${lastName}`;
+console.log(`value of fullName:`, fullName);
+console.log(`type of fullName:`, typeof fullName);
+const pH = 6;
+let isAcidic;
+pH < 7 ? (isAcidic = true) : (isAcidic = false);
```

Comment:
Nice job implementing a ternary! We don't even cover these in the prep course, so good job figuring that out.

This works, but what we actually were going for here is this `const isAcidic = pH < 7;`. We are trying to showcase that anytime you use a comparison operator such as (<) the result will always evaluate to a boolean. So this one line will handle both assigning `true` or `false` depending on the condition.